.docker/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '2.4'
```

Comment:
```suggestion
version: '3.9'
```

Reply:
This is based on the version Skyra uses

https://github.com/skyra-project/skyra/blob/main/.docker/docker-compose.yml#L1